pkg/reconciler/pipelinerun/resources/pipelineref_test.go

```diff
@@ -636,15 +640,16 @@ func TestGetPipelineFunc_VerifyNoError(t *testing.T) {
 			},
 			Spec: signedPipeline.Spec,
 		},
-		expectedRefSource: noMatchPolicyRefSource,
+		expectedRefSource:          noMatchPolicyRefSource,
+		expectedVerificationResult: nil,
 	},
 	}
 	for _, tc := range testcases {
 		t.Run(tc.name, func(t *testing.T) {
 			ctx = test.SetupTrustedResourceConfig(ctx, tc.verificationNoMatchPolicy)
 			fn := resources.GetPipelineFunc(ctx, k8sclient, tektonclient, tc.requester, &tc.pipelinerun, tc.policies)
 
-			resolvedPipeline, source, err := fn(ctx, pipelineRef.Name)
+			resolvedPipeline, source, vr, err := fn(ctx, pipelineRef.Name)
```

Comment:
https://google.github.io/styleguide/go/decisions#got-before-want

```suggestion
			gotPipeline, gotSource, gotVerificationResult, gotErr := fn(ctx, pipelineRef.Name)
```

Reply:
updated